tests/integration/test_estmethod.py

```diff
@@ -5,10 +5,12 @@
 
 
 @pytest.mark.parametrize(
-    'algorithm, methods, solvers, no_of_candidates, advan_ref',
+    'algorithm, methods, solvers, covs, no_of_candidates, advan_ref',
     [
-        ('exhaustive', ['foce', 'imp'], None, 2, 'ADVAN2'),
-        ('exhaustive_only_eval', ['foce', 'imp'], None, 2, 'ADVAN2'),
+        ('exhaustive', ['foce', 'imp'], None, None, 2, 'ADVAN2'),
+        ('exhaustive_only_eval', ['foce', 'imp'], None, None, 2, 'ADVAN2'),
+        ('exhaustive', ['foce'], None, ['sandwich', 'cpg'], 2, 'ADVAN2'),
+        ('exhaustive', ['foce', 'imp'], None, ['sandwich', 'cpg'], 4, 'ADVAN2'),
```

Comment:
I would skip this test, these integration tests are quite slow to run. If you want to test number of models you can add some cases to this test: https://github.com/pharmpy/pharmpy/blob/f357a01c2242be765dc93ddd7b94358d153f88fd/tests/tools/test_estmethod.py#L7-L25